web/services/data_processing_service.py

```
@@ -0,0 +1,318 @@
+            "shareholder_info": "주주에 관한 사항",
+            "investor_protection": "그 밖에 투자자 보호를 위하여 필요한 사항",
+            "contingent_liabilities": "우발부채 등에 관한 사항"
+        }
+
+        for key, title in target_map.items():
+            target_doc = sub_docs[sub_docs['title'].str.contains(title)]
+            if not target_doc.empty:
+                try:
+                    target_url = target_doc['url'].values[0]
+                    headers = {'User-Agent': self.ua.random, 'Referer': 'https://dart.fss.or.kr/'}
+                    response = requests.get(target_url, headers=headers)
```

Ruff (0.14.7)

Probable use of `requests` call without timeout

(S113)